hybridse/src/vm/core_api.h

```diff
@@ -170,6 +170,12 @@ class CoreAPI {
             const int inputRowSizeInBytes, const bool is_instance,
             size_t append_slices, WindowInterface* window);
 
+    static hybridse::codec::Row UnsafeWindowProjectBytes(
```

Comment:
can `UnsafeWindowProjectDirect` be deleted

Reply:
Yes and we will refactor to remove the unused function and unused parameters.